rest-starter/src/main/resources/META-INF/archetype-post-generate.groovy

```diff
@@ -8,13 +10,14 @@ switch (request.properties["runtime"])
 {
     case "glassfish": println "Generating code for GlassFish"
                       FileUtils.forceDelete(new File(outputDirectory, "Dockerfile"))
-                      if ( jakartaVersion != '8')  throw new RuntimeException("Jakarta version not available for this runtime yet")
                       break
                       
     case "tomee":     println "Generating code for TomEE"
-                      if ( jakartaVersion != '8')  throw new RuntimeException("Jakarta version not available for this runtime yet")
-                      break                      
-                      
+                      if ( jakartaVersion != '8') {
```

Comment:
Hey, @jeyvison, shouldn't we specify the version instead of the wildcard, like what will happen when we introduce 10?

Reply:
Hi @rokon12 . We probably could do it but i'm assuming that the chances that , for example in this case, we have an arquillian/Tomee implementation for Jakarta 10 is way lower than we have  for jakarta 9( the reasion why i'm excluding the test package), therefore i dont think we should worry to much about it.

Reply:
We want to include unit test cases for all runtime though!

Reply:
I agree, but all the tests we have are integrations tests though. Are you suggesting creating specific unit tests for tomee?

Reply:
Also, can we please move the discussion to here https://github.com/eclipse-ee4j/starter/pull/119?